app/client/reducers/rootReducer.ts

```diff
@@ -27,10 +27,53 @@ import { OrganisationReducer } from "./organisation/reducers";
 import { metrics } from "./metric/reducers";
 import { allowedFilters } from "./allowedFilters/reducers";
 import { tokens } from "./token/reducers";
+import { all } from "redux-saga/effects";
+import { userSchema } from "../schemas";
+import {
+  createReducers,
+  createSagas,
+  Registration,
+  Body
+} from "../genericState";
+import { RegistrationMapping } from "../genericState/types";
 
 //might be because of older version of react-redux that have to force like this...
 const form = <Reducer<FormStateMap, AnyAction>>FormReducer;
 
+interface CreateUserBody {
+  first_name: string;
+  last_name: string;
+  public_serial_number: string;
+  //  And so on
+}
+
+interface ModifyUserBody {
+  //Defaults to the same as create body, but can be different
+  first_name: string;
+  last_name: string;
+  some_other_thing: boolean;
+  //  And so on
+}
```

Comment:
So assuming this would just be imported from the specific types folder, i.e. `reducer/user/types.tsx`

Reply:
yep. Or at least a file.